src/Controller/DashboardController.java

```diff
@@ -339,5 +347,24 @@ else if(door == doorsTop.get(0) || doorsTop.size() > 1 && door == doorsTop.get(1
 		gc.setStroke(Color.BLACK);
 		gc.fillText(room.getName(), x + 5, y + 17);
     }
-    
+
+    //shows entered date in label box
+	public void displayDate(javafx.event.ActionEvent actionEvent) {
```

Comment:
I just realized that these two methods do not update the actual date and time parameters of the simulation class.  I would look at Mahad's createUser() method for inspiration.  Essentially it just calls an associated method within the Simulation class that has all the logic within it.   That returns a string which will be the  result message thats printed to console.  Then the updateDashboard() is the method that actually changes the label values from the values that are in the simulation class.

Reply:
yeah so I everything works but as I sad before we should set the data and time parameters for the Simulation class.  Also this logic should be moved inside methods of the simulation class.  Look at the createUser() method or my set-house-location branch (which has a PR open) for inspiration.